sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -316,6 +321,25 @@ class RocksDB(
     }
   }
 
+  def merge(key: Array[Byte], value: Array[Byte],
+            colFamilyName: String = StateStore.DEFAULT_COL_FAMILY_NAME): Unit = {
```

Comment:
Also maybe change the indent for all args to be 4 spaces and on new lines similar to functions above ?

Reply:
Done.